library/alloc/tests/vec.rs

```diff
@@ -1594,7 +1594,7 @@ fn partialeq_vec_and_prim() {
 }
 
 macro_rules! assert_partial_eq_valid {
-    ($a2:ident, $a3:ident; $b2:ident, $b3: ident) => {
+    ($a2:expr, $a3:expr; $b2:expr, $b3: expr) => {
```

Comment:
```suggestion
    ($a2:expr,$a3:expr; $b2:expr,$b3: expr) => {
```
We could make it similar to how it was written.